README.md

```diff
@@ -1,128 +1,108 @@
+You can use the CLI to spin up a local emulator of the Griptape Cloud Skatepark.
+Use the Skatepark emulator to develop, test, and validate that your program will operate as expected when deployed as a Griptape Cloud Manage Structure.
+
+1. Start by creating a new repository in your own Github account from the [Managed Structure Template](https://github.com/griptape-ai/managed-structure-template).
+    a. Make sure you're logged in to GitHub.
+    b. Go to the [Managed Structure Template repo](https://github.com/griptape-ai/managed-structure-template).
+    c. Select the *Use this template* drop-down.
+    d. Choose *Create a new repository*.
+    e. Provide a name for the new repository and (optionally) a description.
+    f. Press the *Create repository* button.
+    g. You now have a repository in your own GitHub account that is a copy of the Managed Structure Template for you to begin working with.
+2. Clone your newly-created repository to a directory in your local development environment so that you can begin authoring your own Griptape Managed Structure.
+3. Start the Skatepark emulator.
+    ```bash
+    gt skatepark start
+    ```
+4. In a separate terminal window, register a Structure from within the directory of the locally cloned repository.
+    ```bash
+    gt skatepark register --directory structure/ --main-file structure.py
+    ```
+    This will result in the ID of the registered Structure. It is important to make note of this ID as it will be used to distinguish which Structure you want to run. You can register any number of Structures.
+
+    The example client program uses the environment variable `GT_STRUCTURE_ID` to determine which Structure to run.
+    Set this environment variable to the Structure ID you registered in the previous step.
+    ```bash
+    export GT_STRUCTURE_ID={STRUCTURE_ID}
+    ```
+
+    Or you can register and set the environment variable in one step.
+    ```bash
+    export GT_STRUCTURE_ID=$(gt skatepark register --directory structure/ --main-file structure.py)
+    ```
+
+> [!IMPORTANT]
+> Structures registered with the emulator are not persisted across restarts. You will need to re-register the Structure each time you restart the emulator.
+5. Confirm that the Structure is registered.
+    ```bash
+    gt skatepark list
+    ```
+    You should see a list of registered Structures and the directories they point to, confirming that your Managed Structure was properly registered
+6. You can load environment variables into your Structure by creating an `.env` file in the directory of the Structure you registered. 
+    a. Create a file named `.env` in the `structure/` directory.
+    b. Open the `.env` file in a text editor
+    c. The template expects an `OPENAI_API_KEY` environment variable by default to function. Add OPENAI_API_KEY=_your OpenAI API Key here_ to the `.env` file and save it.
+    d. Add any other environment variables your Structure depends on as you expand the template.
```

Comment:
My apologies for wording last night, I now thing "As you expand on the template, you may add any other environment variables your Structure depends on to this file."